tox.ini

```diff
@@ -104,7 +104,8 @@ deps =
     -e .
     -e testing
 commands =
-    pytest -n auto --ignore={[vars]tst_path}smoke -v --tb native \
+    pytest -n auto --ignore={[vars]tst_path}smoke --ignore={[vars]tst_path}test_benchmark.py \
+        -v --tb native \
```

Comment:
Does pytest run tests in directories recursively by default? If not, would it be better to put these in a `test/benchmark/` directory to avoid having to ignore them in various places and keep them a bit more separate? It may also avoid the pages of "Skipping non-benchmark" output.

Reply:
> Does pytest run tests in directories recursively by default? If not, would it be better to put these in a `test/benchmark/` directory to avoid having to ignore them in various places and keep them a bit more separate? 

Yeah, I wasn't sure where the best place for this was. I'm happy to move it to a separate directory - the tox command can point there specifically so any recursing wouldn't be an issue.

> It may also avoid the pages of "Skipping non-benchmark" output.

Yeah, definitely not a fan of those.